src/web/client/extension.ts

```diff
@@ -187,6 +191,13 @@ export function activate(context: vscode.ExtensionContext): void {
     showWalkthrough(context, WebExtensionContext.telemetry);
 }
 
+export function powerPagesNavigation() {
+    const powerPagesNavigationProvider = new PowerPagesNavigationProvider();
+    vscode.window.registerTreeDataProvider('powerpages.powerPagesFileExplorer', powerPagesNavigationProvider);
```

Comment:
Is disposal of commands required in web extensions? Like use of context.subscriptions.push()

Reply:
Ideally we should - taking this as a separte task to be addressed in next PR.